packages/react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -244,7 +244,7 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef<
       <div role="toolbar" {...divProps} ref={ref}>
         <FocusZone
           componentRef={focusZoneRef}
-          direction={FocusZoneDirection.horizontal}
+          direction={focusZoneDirection}
```

Comment:
You should remove `FocusZoneDirection` from the imports above since it's not being used in this file anymore.

Reply:
@khamudom  I reverted direction back to horizontal as it was before and spread focusZoneProps object if somebody need to override.

Reply:
> @khamudom I reverted direction back to horizontal as it was before and spread focusZoneProps object if somebody need to override.

hi @nikolenkoanton92 did mean to tag me on this?

Reply:
tagging @khmakoto